_posts/2021-03-31-relaxing-requirements-of-moved-from-objects.md

```diff
@@ -139,24 +152,29 @@ for (std::string line; std::getline(std::cin, line);) {
 }
 ```
 
-For the call to `std::getline()` to be valid with a moved from string it requires that `std::string()` _guarantees_ that `std::erase()` is valid on a moved from string. Changing the requirements of a moved from object does not change the guarantees of the standard components.
+For the call to `std::getline()` to be valid with a moved-from string, `std::string()` must _guarantee_ that `std::erase()` is valid on a moved-from string. <!-- I don't understand the assertion.  How is erase involved? --> Changing the requirements of a moved-from object does not change the guarantees of the standard components.
```

Comment:
`std::getline()` is defined to call `str.erase()` in this case - I should make the connection more clear. This section was added as a response to complaints that it wouldn't be valid with my proposed wording for `getline()` to call `erase()`.

Reply:
OK, feel free to hit the “resolve” button when you no longer need a reminder.